source/ui/BioXAS/BioXASPersistentView.cpp

```diff
@@ -38,6 +38,13 @@ BioXASPersistentView::BioXASPersistentView(QWidget *parent) :
 
 	setLayout(layout);
 
+	// Create SR1 current view.
+
+	BioXASControlEditor *sr1CurrentEditor = new BioXASControlEditor(CLSStorageRing::storageRing()->ringCurrentControl());
+	sr1CurrentEditor->setTitle("SR1 Current");
```

Comment:
I'm not sure there needs to be an explicit setting in this case. The ring current control is a read-only PV control in CLSStorageRing, and the control editor checks whether the control can move before displaying the 'edit dialog'. If the control can't move, the editor is effectively read-only. Is the worry that read-only PV controls may inappropriately report as being able to move?